apl-core/src/main/java/com/apollocurrency/aplwallet/apl/core/http/get/GetPollVote.java

```diff
@@ -28,14 +28,17 @@
 import com.apollocurrency.aplwallet.apl.core.http.HttpParameterParserUtil;
 import com.apollocurrency.aplwallet.apl.core.http.JSONData;
 import com.apollocurrency.aplwallet.apl.core.app.AplException;
+import com.apollocurrency.aplwallet.apl.core.service.state.PollService;
 import com.apollocurrency.aplwallet.apl.util.JSON;
 import org.json.simple.JSONStreamAware;
 
 import javax.enterprise.inject.Vetoed;
+import javax.enterprise.inject.spi.CDI;
 import javax.servlet.http.HttpServletRequest;
 
 @Vetoed
 public class GetPollVote extends AbstractAPIRequestHandler {
+    private static final PollService pollService = CDI.current().select(PollService.class).get();
```

Comment:
You can put that into : AbstractAPIRequestHandler for common usage in different places / handlers.